airflow/cli/cli_config.py

```diff
@@ -1754,6 +1754,12 @@ class GroupCommand(NamedTuple):
         func=lazy_load_command("airflow.cli.commands.provider_command.hooks_list"),
         args=(ARG_OUTPUT, ARG_VERBOSE),
     ),
+    ActionCommand(
+        name="triggers",
+        help="List registered provider triggers",
+        func=lazy_load_command("airflow.cli.commands.provider_command.triggers_list"),
+        args=(ARG_OUTPUT, ARG_VERBOSE),
```

Comment:
Would it make sense to provide an additional optional argument for provider names to filter the triggers by provider name as the list could grow long?

Reply:
If we do, we should add it for all other provider commands in a separate PR - this should be consistent across all providers CLI commands.

Reply:
okay, thanks. I will take that up in a separate PR.